packages/types/src/lib/interfaces.ts

```diff
@@ -1178,6 +1178,8 @@ export interface LitAuthClientOptions {
   debug?: boolean;
 
   litOtpConfig?: OtpProviderOptions;
+
+  litNetwork?: LIT_NETWORKS_KEYS;
```

Comment:
Would it make sense to make this a required field?

Anytime you don't provide a network, the consuming function would have to default it (which is probably not the behaviour we want).

Reply:
I think it should be optional, because if they pass in `litNodeClient` they don't need to provide `litNetwork` because it's already set in the `litNodeClient`